cargo-apk/README.md

```diff
@@ -73,6 +73,25 @@ version = 1
 name = "android.permission.WRITE_EXTERNAL_STORAGE"
 max_sdk_version = 18
 
+# See https://developer.android.com/guide/topics/manifest/uses-permission-element
+#
+# Note: there can be multiple .uses_permission entries.
+[[package.metadata.android.uses_permission]]
+name = "org.khronos.openxr.permission.OPENXR"
+
+# See https://developer.android.com/guide/topics/manifest/uses-permission-element
+#
+# Note: there can be multiple .uses_permission entries.
+[[package.metadata.android.uses_permission]]
+name = "org.khronos.openxr.permission.OPENXR_SYSTEM"
+
+# See https://developer.android.com/guide/topics/manifest/queries-element
+#
+# Note: `name` attribute is not required for queries-provider however this is a workaround for aapt throwing errors about missing `android:name` attributes.
```

Comment:
I did this initially but replicated all the other examples that put notes this way.

Reply:
Done.

Reply:
> I did this initially but replicated all the other examples that put notes this way.

The other notes here apply directly to the element (as of writing always an array element enclosed in `[[..]]`) on the line directly following.

Reply:
Note that the suggestion here also re-wraps the comment across two lines, and adjusts the grammar a bit.

Reply:
I can't accept the suggested commit.

Reply:
@korejan I guess you'll have to copypaste it then. Hovering over the buttons says: "outdated suggestions cannot be applied" because these lines and/or the context around it changed since it was posted.